recipes/survindel2/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "survindel2" %}
+{% set version = "1.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/kensung-lab/SurVIndel2/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 8463d4b5f5060f26520d8a57f162f74708e0a3ecb68675dec6af671688a21a3f
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('survindel2', max_pin='x.x') }}
+  skip: True  # [osx]
+
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - htslib >=1.13
+  run:
+    - pysam >=0.16.0.1
+    - pyfaidx >=0.5.9.1
+    - numpy
+    - scikit-learn >=1.2.2
+
+test:
+  source_files:
+    - demo/reads.bam
+    - demo/reads.bam.bai
+    - demo/ref.fa
+    - demo/ref.fa.fai
+
```

Comment:
Can you maybe add something like `reads_categorizer --help`

Reply:
reads_categorizer does not have a --help argument. The executables are not meant to be used directly, but they are used by survindel2.py.